pvlib/iotools/psm3.py

```diff
@@ -20,12 +21,31 @@
     'surface_pressure', 'wind_direction', 'wind_speed')
 PVLIB_PYTHON = 'pvlib python'
 
+# Dictionary mapping PSM3 names to pvlib names
+PSM3_VARIABLE_MAP = {
```

Comment:
Shall we change this to just `VARIABLE_MAP`?  https://github.com/pvlib/pvlib-python/pull/1368#discussion_r782620946